src/components/atoms/SocialMedia.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import LinkedinIcon from '../.././assets/icons/linkedin-icon.svg';
+import TelegramIcon from '../.././assets/icons/telegram-icon.svg';
+import WhatsAppIcon from '../.././assets/icons/whatsapp-icon.svg';
+
+const SocialMedia = () => {
+  return (
+    <div className="flex flex-row gap-x-5">
```

Comment:
let's extract this <link><imgage></link> logic into its own component, call it socialIcon

Reply:
Done 👍